### DISCUSSION TITLE
Save live timing screen(s) location as part of saved layout

### DISCUSSION THREAD

**Discussion by meph0** [f1multiviewer/issue-tracker#266] [Feature requests] [answered]
I need to open, resize and re-position the live map and race control every session. I'd love for everything to open as part of my saved layout.

**Comment by f1multiviewer** [comment, marked as answer]
Duplicate of #7

**Reply by meph0** [reply]
Apologies, didn't come up with my search terms.

**Reply by f1multiviewer** [reply]
No worries, I improved the title because it was hard to find (but I knew it was there)
